convertAngle.test.ts

```diff
@@ -0,0 +1,19 @@
+import convertAngle from "./convertAngle.ts";
+import { assertEquals, assertThrows } from "testing/asserts.ts";
+
+Deno.test("convertAngle", () => {
+  assertEquals(convertAngle(123, "degree", "degree"), 123);
+  assertEquals(convertAngle(123, "radian", "radian"), 123);
+  assertThrows(() => convertAngle(123, "invalidUnit", "invalidUnit"), 123);
```

Comment:
This test is failing because `convertAngle` doesn't throw when `from === to`.
You can run the tests by running `scripts/test`.

Reply:
Sorry, I copied pasted badly.